modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationHistoryPropsTransformer.js

```diff
@@ -75,10 +79,10 @@ export default function propsTransformer({
 						}
 					}
 				});
-		}, [props.itemId]);
+		}, [props]);
 
-		if (status) {
-			return <span className="workflow-status">{status.label}</span>;
+		if (props.value.label !== 'in-progress' || status) {
```

Comment:
I had to check the prop after the hook or else I would get `React Hook "useEffect" is called conditionally` errors 😕

Reply:
https://github.com/liferay-publications/liferay-portal/assets/46908843/45d89c6c-dc93-499d-93ce-ecfb30c9a2da